webknossos/webknossos/cli/convert.py

```diff
@@ -102,6 +102,15 @@ def main(
     compress: Annotated[
         bool, typer.Option(help="Enable compression of the target dataset.")
     ] = False,
+    batch_size: Annotated[
+        Optional[int],
+        typer.Option(
+            help="Number of images to be processed in one batch. "
```

Comment:
```suggestion
            help="Number of images to be processed in one batch (influences RAM consumption). "
```

I think the same string is somewhere else? probably adapt there too.